mujoco_warp/_src/io.py

```diff
@@ -659,6 +664,17 @@ def make_data(
     d.qM = wp.zeros((nworld, sizes["nv_pad"], sizes["nv_pad"]), dtype=float)
     d.qLD = wp.zeros((nworld, mjm.nv, mjm.nv), dtype=float)
 
+  if mujoco.mj_isSparse(mjm):
+    d.flexedge_J_rownnz = wp.zeros((nworld, mjm.nflexedge), dtype=int)
```

Comment:
I guess all the dense branches of flexedge_J will be deleted once we make it always sparse in mujoco?